migrations/V68__Remove_Architect_Role_From_Team_Roles.sql

```diff
@@ -0,0 +1,16 @@
+UPDATE plan_collaborator SET team_role = 'IT_LEAD' WHERE team_role = 'ARCHITECT';
+
+ALTER TYPE TEAM_ROLE RENAME TO TEAM_ROLE_OLD;
+
+CREATE TYPE TEAM_ROLE AS ENUM (
+    'MODEL_LEAD',
+    'MODEL_TEAM',
+    'LEADERSHIP',
+    'LEARNING',
+    'EVALUATION',
+    'IT_LEAD'
+);
+
+ALTER TABLE plan_collaborator ALTER COLUMN team_role TYPE TEAM_ROLE USING team_role::TEXT::TEAM_ROLE;
```

Comment:
Ahh - good call!

Reply:
Thanks Clay! In case we ever look back here at the resolution, besides this, you can't alter a column type if a trigger depends on it. You have to drop the triggers and re-add them to make it work.